content/tutorials/aws-persistence.textile

```diff
@@ -0,0 +1,249 @@
+---
+authors:
+- author_bio: ''
+  author_image: 
+  author_name: Steven Applebly
+  author_profile_url: 
+category:
+- integrations
+date_published: '2016-07-22T02:38:02+01:00'
+excerpt: Learn how to persist messages into AWS S3.
+group: sdk
+index: 10
+languages:
+- javascript
+last_updated: '2021-11-19T16:35:45+00:00'
+level: easy
+platform: mixed
+reading_time: 5 — 10
+section: tutorials
+tags:
+- Integrations
+- AWS Lambda
+title: AWS Persistence in S3
+meta_description: "How to persist data into S3."
+meta_keywords: "Ably realtime, AWS Lambda Functions, integrations"
+---
+
+Ably "webhooks":/general/events enable users to configure custom rules that react to channel events. For example, triggering an action when a message is sent, or when a user joins the channel. You can use our integration with "AWS Lambda":https://aws.amazon.com/lambda/ to create these rules.
+
+In this tutorial you will learn how to create a channel rule that intercepts messages sent to a channel and persists them to a file in "AWS Simple Storage Service (S3)":https://aws.amazon.com/s3/.
+
+###. TODO: update the GH repo link
+
+The solution code for this tutorial is "available on Github":https://github.com/ably/tutorials/TODO
```

Comment:
Once the code is in its own repo we can fix this link. I would avoid using the tutorials repo, and follow the guidelines set by the other two AWS tutorials, and use a separate repo for this example code.